Abcbadq/sjsu_shop
# SJSU-1001, Finish Register & Login, Deploying to heroku

_ Backend and frontend for login & register feature
_ Landing Page for SpartanStop
_ Setups for deploying to Heroku
_ Verification Steps:
       + go to https://peaceful-mountain-86940.herokuapp.com/
       + update the latest master and run the code
       + register for new user account
       + login with new user account
       + should be directed to landing page after logged in 
       + logout by clicking button at the top right corner
